src/tapis-ui/_common/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -29,7 +29,7 @@ const Breadcrumbs: React.FC<BreadcrumbsProps> = ({ breadcrumbs }) => {
   return (
     <div>
       {breadcrumbs.map((item, index) => {
-        return index === breadcrumbs.length - 1 || item.to === undefined ? (
+        return index === breadcrumbs.length - 1 || !item.to ? (
           <span> {item.text} /</span>
```

Comment:
the spans require keys property. maybe use uuidv4

Reply:
That what I was thinking but hated to go that route. Oh well. uuidv4 it is